Library/Homebrew/requirements/cctools_requirement.rb

```diff
@@ -0,0 +1,8 @@
+class CctoolsRequirement < Requirement
+  fatal true
+  default_formula 'cctools'
+
+  satisfy do
+    MacOS::Xcode.installed? || MacOS::CLT.installed?
```

Comment:
Shouldn't this be `MacOS::Xcode.installed? || MacOS::CLT.installed? || Formula["cctools"].installed?`